web/src/main/webapp/WEB-INF/jsp/managestudy/studySubject/casebookGenerationForm.jsp

```diff
@@ -6,54 +6,93 @@
-                               name="GetLink">
-                        <input type="button" id="casebookOpenBtn" class="button_medium"
-                               value="<fmt:message key="viewStudySubject.casebookGenerationForm.button.open" bundle="${resword}"/>"
-                               name="exit">
-                    </div>
-                    <div style="display:none" id="casebookLinkDisplay"><br>
-                        <fmt:message key="viewStudySubject.casebookGenerationForm.button.getLink.description"
-                                     bundle="${resword}"/><br>
-                        <input type="text" id="casebookLinkText" size="100"></div>
-                </div>
-            </form>
+            <div style="margin-bottom:10px;" class="connect-widget">
 
+                <div id="casebookButtons">
+                    <input type="button" id="casebookLinkBtn" class="button_medium"
+                           value="<fmt:message key="viewStudySubject.casebookGenerationForm.button.getLink" bundle="${resword}"/>"
+                           name="GetLink">
+                    <input type="button" id="casebookOpenBtn" class="button_medium"
+                           value="<fmt:message key="viewStudySubject.casebookGenerationForm.button.open" bundle="${resword}"/>"
+                           name="exit">
+                    <input type="button" id="casebookPdfBtn" class="button_medium"
+                           value="<fmt:message key="viewStudySubject.casebookGenerationForm.button.getPdf" bundle="${resword}"/>"
+                           name="GetPDF" disabled="disabled">
+                </div>
+                <div style="display:none" id="casebookLinkDisplay"><br>
+                    <fmt:message key="viewStudySubject.casebookGenerationForm.button.getLink.description"
+                                 bundle="${resword}"/><br>
+                    <input type="text" id="casebookLinkText" size="100"></div>
+            </div>
         </div>
-
     </div>
-
 </div>
-<!-- casebookGenerationForm.jsp - END -->
\ No newline at end of file
+<!-- casebookGenerationForm.jsp - END -->
+
+<script>
+    $('input[type=radio][name=casebookType]').change(function() {
+        if ($(this).val() === 'pdf/view') {
+            $('#casebookOptions').slideUp();
+            $('#casebookLinkBtn, #casebookOpenBtn').attr('disabled', 'disabled');
+            $('#casebookPdfBtn').removeAttr('disabled');
+        }
+        else {
+            $('#casebookOptions').slideDown();
+            $('#casebookLinkBtn, #casebookOpenBtn').removeAttr('disabled');
+            $('#casebookPdfBtn').attr('disabled', 'disabled');
+        }
+    });
+
+    $('#casebookPdfBtn').click(function () {
+        $('#getPdfMessage').slideDown();
+        jQuery.ajax({
+            'type': 'POST',
+            'url': '${pageContext.request.contextPath}/auth/api/clinicaldata/pdf/print/${study.oid}/${studySub.oid}?format=A4&landscape=false&margin=0.5in',
```

Comment:
don't set "?format=A4&landscape=false&margin=0.5in" since these defaults are set in the backend code

Reply:
also Tao is going to change the url to "/studies/{studyOID}/participants/{participantId}/casebook" so please update this accordingly

Reply:
@svadla-oc OK removed and updated